paymentsheet/src/test/java/com/stripe/android/paymentsheet/state/DefaultPaymentSheetLoaderTest.kt

```diff
@@ -1467,6 +1467,60 @@ internal class DefaultPaymentSheetLoaderTest {
         assertThat(observedElements).containsExactlyElementsIn(expectedElements).inOrder()
     }
 
+    @OptIn(ExperimentalCustomerSessionApi::class)
+    @Test
+    fun `When using 'CustomerSession', payment methods should be filtered by supported saved payment methods`() =
```

Comment:
can you explicitly set the supported saved payment methods somewhere in this test? If we changed that list in the future to e.g. include link then this test would start failing and I think it could be hard to figure out how to fix it.

Also, can you out some of the details in this test that aren't related to what's being tested (like the customer session fake data) so that it's easier to tell what's being tested?

Reply:
it looks like this was addressed in the `CustomerStateTest`. Do we still need this test? If yes, are you able to update this one too?

Reply:
Unfortunately `PaymentMethodMetadata` hardcodes the payment method types directly from the `PaymentMethodRegistry`. I think it's still worth keeping this test in case that changes.